e2e/hook_test.go

```diff
@@ -142,77 +139,4 @@ func testHook() {
 			Expect(hasFinalizer).Should(Equal(true), "finalizer is not set: pvc=%s", pvc.Name)
 		}
 	})
-	It("should test hooks for inline ephemeral volumes", func() {
```

Comment:
Don't you need to replace it with the test of generic ephemeral volume?

Reply:
I added a test case.